.gitignore

```diff
@@ -1,3 +1,3 @@
 /target/
 .*
-
+*.iml
```

Comment:
Oh, I see. I thought it was common to add well-known IDE extensions to .gitignore?